ako_cicd/jenkins/ci-build/save_build_on_mnt.sh

```diff
@@ -2,12 +2,12 @@
 
 set -xe
 
-cd $WORKSPACE/hack/jenkins;
-echo $GIT_COMMIT > $WORKSPACE/HEAD_COMMIT;
-
 CI_REGISTRY_PATH=$PVT_DOCKER_REGISTRY/$PVT_DOCKER_REPOSITORY
 BRANCH=$branch
 
+echo $(git rev-parse origin/${branch}) > $WORKSPACE/HEAD_COMMIT;
+cat $WORKSPACE/HEAD_COMMIT
```

Comment:
is `cat` of HEAD_commit required?

Reply:
yes , we need this, it will be referred in jenkins job level